examples/tests/invalid/bring.test.w

```diff
@@ -4,4 +4,8 @@ bring cloud;
 bring cloud;
 //    ^^^^^ "cloud" is already defined
 bring fs;
-//    ^^^^^ "fs" is not a built-in module
\ No newline at end of file
+//    ^^^^^ "fs" is not a built-in module
+bring ;
+//^^^^^ Expected built-in module or "external module"
```

Comment:
My plan is to reserve the non quoted namespace for "managed" modules from the `winglang/library` repository. See https://github.com/winglang/wing/issues/1037 (copy @Chriscbr).

So "built-in" is likely not the right term. Also, this syntax has a variant of a local module as well 🤦‍♀️

Maybe:

```suggestion
//^^^^^ Expected module specification (see docs/modules).
```